test-cases/tools/cmd/lint.ts

```diff
@@ -0,0 +1,297 @@
+import * as path from "path";
+import { CommandModule } from "yargs";
+import {
+    AUTOMATED_TAG,
+    DESTRUCTIVE_TAG,
+    MANUAL_SELECTION_TAG,
+    PER_BUILD_TAG,
+    PER_RELEASE_TAG,
+    STEPS_SECTION,
+} from "../lib/constants";
+import {
+    desiredFileName,
+    isAutomated,
+    isPerBuild,
+    isPerRelease,
+    loadTestCases,
+    manualSelectionOnly,
+    TestCase,
+} from "../lib/test-case";
+import { isEmpty } from "../lib/utils";
+import { logger } from "../lib/winston";
+
+type Linter = (test: TestCase) => error;
+
+type error = string | null;
+
+const AUTOMATION = /^[A-Z]+-[0-9]+$/;
+
+const CATEGORIES = [
+    "alerts",
+    "authorization",
+    "backup-restore",
+    "dashboards",
+    "documentation",
+    "high-availability",
+    "installation",
+    "monitoring",
+    "performance",
+    "products",
+    "upgrade",
+    "walkthroughs",
+    "uninstallation",
+];
+
+// Update the README.md too
```

Comment:
@b1zzu readme updated, can be removed

Reply:
few other occurrences below can be removed too.

Reply:
This is more a reminder that if you add a new category, tag or component you should also update also mention it in the README.md because they are all documented there

Reply:
I should update the comment to explain it better